components/library/libtdjson/Makefile

```diff
@@ -0,0 +1,74 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 David Stes
+#
+
+# currently build with gcc-11 instead of gcc-7 (out of memory with gcc-7)
+# Tdlib provides a php script to SplitSources which may help in the gcc-7 case
+
+# 1.8.0 is easier to build than 1.6.0
+# the Illumos and Solaris platforms were added in 1.7.0 or 1.8.0 to config.h
+# and platform.h so they are supported target platforms
+# also see CMake/illumos.cmake in the TdLib sources
+
+# because we currently need 1.6.0 in Squeak, the CMakeLists.txt
+# config.h and platform.h is patched so that 1.6.0 builds (as 1.8.0 does)
+# see www.squeak.org for more information for general info on Squeak
+# also see http://docs.openindiana.org/handbook/community/squeak/index.html
+# the Telegram Squeak Client is at
+# https://github.com/hpi-swa-teaching/TelegramClient
+
+BUILD_BITS=		32_and_64
+BUILD_STYLE=		cmake
+# USE_OPENSSL11=		yes
+include ../../../make-rules/shared-macros.mk
+GCC_VERSION= 		11
+
+# the FMRI is libtdjson , the project is tdlib, component is td
+
```

Comment:
Why this? Wouldn't it be better to set COMPONENT_NAME=tdlib and COMPONENT_FMRI=library/$(COMPONENT_NAME)?